packages/@aws-cdk/aws-ec2/lib/vpc-endpoint-service.ts

```diff
@@ -68,6 +81,8 @@ export class VpcEndpointService extends Resource implements IVpcEndpointService
       acceptanceRequired: this.acceptanceRequired
     });
 
+    this.vpcEndpointServiceId = this.endpointService.ref;
+    this.serviceName = Fn.join('.', ['com.amazonaws.vpce', Aws.REGION, this.vpcEndpointServiceId]);
```

Comment:
And how about in the `aws-cn` partition?

Reply:
Do you know what's wrong with this code that doesn't work for China regions? I'm not familiar with the subjet, and haven't found anything helpful after searching the docs.

Reply:
From cursory internal code searches, seems like it needs to be `cn.com.amazonaws.vpce` (assuming those are correct).

The Right(tm) thing to do would be to add this as a fact to the `@aws-cdk/region-info` package.

Reply:
Would this also be a problem for aws-us-gov?

Reply:
Yap.